docs/userguide.md

```diff
@@ -305,6 +305,27 @@ To cleanup the example simply delete the namespace:
 kubectl delete ns test-namespace
 ```
 
+### Troubleshooting
+
+If you found the federated resources were not propagated to the member clusters, you can always
+use the following command to get some failure reason by checkig the `Events`.
+
+```bash
+kubectl describe <federate CRD> <CR> -n test-namespace
+```
+
+One example for CRD of `federatedserviceaccounts` is as follows:
+
+```bash
+kubectl describe federatedserviceaccounts test-serviceaccount -n test-namespace
+```
+
+If you cannot get too much help info by above command, you can check federation controller log as follows:
```

Comment:
s/.../It may also be useful to inspect the federation controller log as follows: